translations/de.yaml

```
@@ -82,8 +82,8 @@ emeis:
       add-entry: "Eintrag hinzufügen"
 
   acl-wizzard:
-    add-entry: "ACL Eintrag hinzufügen"
-    duplicate: "Diese ACL Kombination existiert bereits. ACL Einträge sind eindeutig."
+    add-entry: "Berechtigung hinzufügen"
+    duplicate: "Diese Kombination der Berechtigungen existiert bereits. Berechtigungen sind eindeutig."
```

I'd be confused by this. How about
```suggestion
    duplicate: "Diese Kombination von Berechtigungen existiert bereits."
```

---

Fixed